cogs/moderator.py

```diff
@@ -65,7 +65,7 @@ def cog_unload(self):
     @command(help="Verify a mentioned user")
     @bot_chat(bot_channels)
     @has_role_plus(moderator)
-    async def verify(self, ctx: Context, member):  # parameter only for pretty help.
+    async def verify(self, ctx: Context, member: int):
```

Comment:
Revert the Type def. To test but I think like this the Command won't work anymore if a mention is given